client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -686,9 +683,7 @@ export default connect(
 				-1,
 			user: getCurrentUser( state ),
 			userDate: getCurrentUserDate( state ),
-			transferComplete:
-				transferStates.COMPLETED ===
-				get( getAtomicTransfer( state, siteId ), 'status', transferStates.PENDING ),
+			transferComplete: transferStates.COMPLETED === getAtomicTransfer( state, siteId ).status,
```

Comment:
Confirming this is fine, `getAtomicTransfer( state, siteId ).status` may at worst return `undefined` but won't throw a TypeError or anything.

Reply:
Yes, the result of `getAtomicTransfer` defaults to an `{}` empty object if the state is not found.